submissions/OlStaniev/OOP exercise/index.js

```diff
@@ -0,0 +1,42 @@
+class Inhabitants {
+   constructor(name, gender, legs, saying) {
+      this.name = name;
+      this.gender = gender;
+      this.legs = legs;
+      this.saying = saying
+   }
+
+   prepareToPrint() {
+      return [this.name, this.gender, this.legs, this.saying].join(';')
+   }
+}
+
+class People extends Inhabitants {
+   constructor(name, gender, legs, hands, saying) {
+      super(name, gender, legs, saying)
+      this.hands = hands
+   }
+   prepareToPrint() {
+      return [this.name, this.gender, this.legs, this.hands, this.saying].join(';')
```

Comment:
There is way to re-use parent class method. It makes sense as this helps to follow SOLID principles. Check the supplementary materials on the task's README.